examples/cookbook/testing/integration/profiling/test_driver/perf_driver.dart

```diff
@@ -0,0 +1,29 @@
+// #docregion Timeline
+import 'package:integration_test/integration_test_driver.dart';
+import 'package:flutter_driver/flutter_driver.dart' as driver;
+
+Future<void> main() {
+  return integrationDriver(
+    responseDataCallback: (data) async {
+      if (data != null) {
+        final timeline = driver.Timeline.fromJson(data['timeline']);
+
+        // Convert the Timeline into a TimelineSummary that's easier to
+        // read and understand.
+        final summary = driver.TimelineSummary.summarize(timeline);
+
+        // Then, write the entire timeline to disk in a json format.
+        // This file can be opened in the Chrome browser's tracing tools
+        // found by navigating to chrome://tracing.
+        // Optionally, save the summary to disk by setting includeSummary
+        // to true
```

Comment:
This looks great. I'm wondering why [the sample in flutter/samples](https://github.com/flutter/samples/blob/master/testing_app/integration_test/perf_driver.dart) uses `data.entries` but your example uses `data.timeline`  - is it because there's [more than one test](https://github.com/flutter/samples/blob/master/testing_app/integration_test/perf_test.dart)? Dealing with these traces has been tricky in the past.

Reply:
yes, good question, the [traceAction method](https://api.flutter.dev/flutter/package-integration_test_integration_test/IntegrationTestWidgetsFlutterBinding/traceAction.html) stores the timeline by default in the `data['timeline']` entry, so if you have more than one perf test, your should change the `reportKey` parameter to something else. I could make the parameter explicit in this example to something like `scrolling_timeline` to avoid confusion.

Reply:
I pushed a change where I specify the `reportKey` instead of using the default one, and added a short explanation about it.